src/nile/core/deploy.py

```diff
@@ -15,8 +26,12 @@ def deploy(
     abi=None,
     mainnet_token=None,
 ):
-    """Deploy StarkNet smart contracts."""
-    logging.info(f"🚀 Deploying {contract_name}")
+    """Deploy StarkNet smart contracts (DEPRECATED)."""
+    logging.info(
+        f"🚀 Deploying {contract_name} without Account. "
+        + "This method is deprecated and will be removed soon. "
+        + "Try using the --account command option."
```

Comment:
```suggestion
        + "Use the --account option."
```
although this is not accurate for nre